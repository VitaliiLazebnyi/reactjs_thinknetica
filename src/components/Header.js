import React from 'react';

function Header() {
  return <div className='logo'>Logo</div>;
}

export default Header;
