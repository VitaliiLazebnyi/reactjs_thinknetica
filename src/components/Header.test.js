import { render, screen } from '@testing-library/react';
import { create } from "react-test-renderer";
import Header from './Header';

describe("Header component", () => {
    test('Renders logo text', () => {
        render(<Header />);
        expect(screen.getByText('Logo')).toBeInTheDocument()
    });

    test('Matches the snapshot', () => {
        const header = create(<Header />);
        expect(header.toJSON()).toMatchSnapshot();
    });
});
