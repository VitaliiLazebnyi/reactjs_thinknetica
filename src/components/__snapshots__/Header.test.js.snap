// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Matches the snapshot 1`] = `
<div
  className="logo"
>
  Logo
</div>
`;
